PickUpApi/orders.ts

```diff
@@ -9,11 +9,6 @@ import { OrderDetailDto } from "./domain/dtos/OrderDetailDto";
 
 
 export class OrdersApi{
-    accountSid = process.env.TWILIO_ACCOUNT_SID;
```

Comment:
so those variables were not used before ? I do not see any update about where they are used.

Reply:
Nevermind, it is on lines 52 , 54